src/odemis/acq/move.py

```diff
@@ -1798,3 +1839,22 @@ def _getCurrentAlignerPositionLabel(self) -> int:
             return LOADING_PATH
         # None of the above -> unknown position
         return UNKNOWN
+
+### utils   
+def get_movement_mode(current_pos: Dict[str, float], target_pos: Dict[str, float]) -> str:
```

Comment:
I've added a more general helper, has_axis_move which will check for a desired movement on the specified axes. I've also added a test for it, because it the previous function has unintuitive default behaviour that needs to be kept for now. 

```python
has_rz: bool = has_axis_move(current_pos, target_pos, "rz", tolerance)
```